evm_loader/program/src/instruction/transaction_execute.rs

```diff
@@ -54,7 +54,13 @@ pub fn execute<'a>(
     let (exit_reason, apply_state) = {
         let mut backend = ExecutorState::new(account_storage);
 
-        let mut evm = Machine::new(trx, caller_address, &mut backend)?;
+        let mut evm = Machine::new(
+            trx,
+            caller_address,
+            &mut backend,
+            #[cfg(feature = "tracing")]
+            None,
+        )?;
```

Comment:
This file should not be compiled when neon-evm is built as a library.
`tracing` feature should not be available when neon-evm is built for Solana

Reply:
The same comment as above. The `tracing` feature is not enabled when `evm-loader` crate is built as a Solana program.

Reply:
Ok, keep it as is. evm_loader crate is currently incorrectly linked to other crates. I will fix this separately.